Apps/AdminWebClient/src/Server/Services/DashboardService.cs

```diff
@@ -129,16 +131,36 @@ public RequestResult<IEnumerable<MessagingVerificationModel>> GetMessageVerifica
                     dbResult = this.messagingVerificationDelegate.GetUserMessageVerifications(Database.Constants.UserQueryType.SMS, queryString);
                     break;
                 case UserQueryType.HDID:
-                    dbResult = this.messagingVerificationDelegate.GetUserMessageVerifications(Database.Constants.UserQueryType.HDID, queryString);
+                    RequestResult<PatientModel> patientResultHdid = Task.Run(async () => await this.patientService.GetPatient(queryString).ConfigureAwait(true)).Result; 
```

Comment:
I believe we might need to return the PHN column for the other types of queries as well.  Will ask Nino to comment.

Reply:
no - let's only return when we are searching with HDID. I don't need to pull PHN for sms/email search.

Reply:
@MikeLyttle Based on the user story, we only show PHN when we search for HDID but I also confirm to Nino that we can show the PHN when we search PHN also.